### ISSUE TITLE
Timeline content is missing headings which can allow screen reader user to quickly jump from one step to another

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4622]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 2.4.6 Headings and Labels

**Test Environment:**
- Test URL: https://clarity.design/documentation/timeline
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 2020.1

**Steps to reproduce:**
1. Open the test URL.
2. Select any one of the tabs provided for horizontal, vertical and angular timelines.
3. Try to perform quick navigation by trying to jump to a particular step.

**Expected result:**
Screen reader user should be able to jump quickly from one step to another.

**Actual result:**
No provision provided to quickly jump through the steps. Note: Visual placement of content in timeline allows the user to segregate the steps, so sighted users and skim-through or jump through the steps quickly. For example, each step is spaced proportiontely under each status symbol and timestamp. Visually some content is marked bold to indicate some section.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
